openid-4-verifiable-presentations-1_0.md

```diff
@@ -1611,17 +1611,30 @@ The signed request allows the Wallet to authenticate the Verifier using a trust
 
 ## Response
 
-Every OpenID4VP Authorization Request MUST result in a response being provided through the W3C Digital Credentials API. The Authorization Response is a JavaScript object, where the response parameters as defined for the Response Type are encoded as top-level members in this JavaScript object. 
+Every OpenID4VP Authorization Request MUST result in a response being provided through the W3C Digital Credentials API. This response is a JavaScript object as defined in [@!w3c.digital_credentials_api], where the OpenID4VP Authorization Response parameters as defined for the Response Type are encoded as members within the `data` member of this JavaScript object.
+
+The following is a non-normative example of an unsigned OpenID4VP response that could be received from the W3C Digital Credentials API:
 
-The following is a non-normative example of an OpenID4VP response that could be received from the W3C Digital Credentials API:
-	
 ```js
 {
-  vp_token: "...",
-  presentation_submission: {...}
+  protocol: "openid4vp",
```

Comment:
A better way to represent this might be as follows, as the returned `DigitalCredential` object is not technically "js object":

```JS
const credential = await navigator.identity.get(request);
console.log(credential.protocol); // "openid4vp"

// Extract relevant data members
const { vp_token, presentation_submission } =  credential.data;

```

Reply:
Do you think it'd be over misleading to show it as an object?

If we go along the lines you suggest I'd probably prefer something a bit closer to how expect developers to use it, perhaps:

```
const credential = await navigator.identity.get(request);
if (credential.protocol == "openid4vp") {
  // Extract relevant data members
  const { vp_token, presentation_submission } =  credential.data;
  // presentation_submission is a javascript object
  // vp_token is a string or javascript object depending on the credential type
```